### PR TITLE
Controlbar opacity fix in audio player, .jw-options styles

### PR DESCRIPTION
**Pull request by kasperific**
Increased opacity in audio player controlbar for controlbars that have <1 opacity. Also fixed wrong colors of cc option list items in five and vapor skins.

### CODE REVIEW

File: src/css/skins/five.less [hunk 1]
```diff
@@ -159,6 +157,17 @@
         }
     }
 
+    .jw-icon-cc {
```
Comment (robwalch) [inline src/css/skins/five.less]: `.jw-icon-cc` should be `jw-icon-tooltip` so that all menus (hd and audiotracks) also get these colors right?

Comment (robwalch) [inline src/css/skins/five.less]: @jw-kaurand can you confirm?

File: src/css/skins/vapor.less [hunk 2]
```diff
@@ -149,7 +148,8 @@
 
         .jw-option {
             color: #fff;
-
+            
+            &:hover,
```
Comment (robwalch) [inline src/css/skins/vapor.less]: @kasperific so now vapor has the same problem where options are only set for cc menu?

**Comment by kasperific** [comment]
Additional commits for making .jw-option styles less specific, without affecting playlist options. Applies to Six, Vapor and Five.

**Pull request merged**